src/auth/providers.js

```diff
@@ -11,6 +11,14 @@ export const PROVIDER_TYPE_GOOGLE = 'google';
 export const PROVIDER_TYPE_FACEBOOK = 'facebook';
 export const PROVIDER_TYPE_MONGODB_CLOUD = 'mongodbCloud';
 
+function urlWithParams(url, link) {
```

Comment:
[nit] This isn't really URL with params, it's URL with link aded on if true. How about `urlWithLinkParam`